muckrock/accounts/models.py

```diff
@@ -158,6 +159,14 @@ def is_advanced(self):
         # pylint: disable=comparison-with-callable
         return self.feature_level > 0
 
+    def sum_pages_for_user(self):
+        # Filter FOIAFile objects for the given user
+        foia_files_for_user = FOIAFile.objects.filter(comm_foia_composer_user=self.user)
```

Comment:
Just noticed this, but these need to be double underscores:
`comm_foia_composer_user`
`comm__foia__composer__user`